backend/btrixcloud/crawlconfigs.py

```diff
@@ -600,6 +600,16 @@ async def stats_recompute_last(self, cid: UUID, size: int, inc_crawls: int = 1):
 
             if last_crawl_finished:
                 update_query["lastRun"] = last_crawl_finished
+        else:
+            update_query["lastCrawlId"] = None
```

Comment:
Hmm, are we sure this is correct?
It will get here if there is a crawl running, since it'll be the else for `if not running_crawl`.
This will then set isCrawlRunning to False while a crawl is running...

I guess the intent previously was to do nothing if crawl is running, as these stats were being updated as part of the running crawl...

Reply:
Hm, oh you're right. As-is it works in the case of running crawls, but this method is also used to handle crawl deletions on a workflow and that's currently broken. When a workflow has crawls but those crawls are all deleted, the statistics are never updated to accurately reflect that there is no last crawl in that case. At one point this worked for that use case and then was modified, probably because of the issue you're pointing out.

Reply:
Pushed a commit which should cover both cases now